WordPress/Classes/Utility/ReachabilityFacade.h

```diff
@@ -0,0 +1,13 @@
+#import <Foundation/Foundation.h>
+
+@protocol ReachabilityFacade <NSObject>
```

Comment:
I imagine this class exists primarily to make mocking easier (by not mocking the static (ie: globally accessible) class `ReachabilityUtils` directly).

I'm a bit torn by this though... it does feel to me like injecting the desired class name in `LoginViewModel`'s `init` method on test runs could be cleaner for our release code?

I don't want to turn this PR into a witch-hunt though, considering how huge it is and that it already brings so much good stuff.  So what I'm saying is feel free to agree/disagree with my review notes in terms of changing these things after the initial merge.